### PR TITLE
update/clean up install commands for recent ubuntu deploy/install on …

### PR DESCRIPTION
**Pull request by tcsiwula**
…new server instance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +46,10 @@ If you missed it you can correct your mistake with command
 
 ### Install CMake
 
+```shell
+sudo scripts/install_cmake.sh --prefix /usr/local
```
Comment (gumb0) [inline README.md]: This command is already there in *Alternative method* section

File: README.md [hunk 2]
```diff
@@ -104,9 +107,9 @@ Configure the project build with the following command. It will create the
 `build` directory with the configuration.
 
 ```shell
-mkdir build; cd build  # Create a build directory.
-cmake ..               # Configure the project.
-cmake --build .        # Build all default targets.
+sudo mkdir build && cd build         # Create a build directory.
```
Comment (gumb0) [inline README.md]: Why exactly do you think sudo is needed?

File: README.md [hunk 3]
```diff
@@ -36,8 +36,7 @@ Windows          | [![AppVeyor](https://img.shields.io/appveyor/ci/ethereum/cpp-
 Git and GitHub is used to maintain the source code. Clone the repository by:
 
 ```shell
-git clone --recursive https://github.com/ethereum/cpp-ethereum.git
-cd cpp-ethereum
+git clone --recursive https://github.com/ethereum/cpp-ethereum.git && cd cpp-ethereum
```
Comment (chfast) [inline README.md]: Making it single line does not make it easier to understand not read.

File: README.md (hunk 2 above)
Comment (chfast) [inline README.md]: The version with `&&` does not work in PowerShell.

**Comment by pirapira** [comment]
@tcsiwula are you trying to continue?

**Comment by tcsiwula** [comment]
@pirapira no, no, can you close this out, please? thanks! 👍

**Comment by pirapira** [comment]
Alright!

**Pull request closed without merging**